extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -106,10 +106,7 @@ export class AmpSelector extends AMP.BaseElement {
     this.init_();
     if (!this.isDisabled_) {
       this.element.addEventListener('click', this.clickHandler_.bind(this));
-      if (this.kbSelectMode_ != KEYBOARD_SELECT_MODES.NONE) {
```

Comment:
the viewport/platform i think has an api to figureout if keyboard is available, in-fact it uses that to add css classes to the body.